app/js/actions/labOrders/labConceptsAction.js

```diff
@@ -0,0 +1,33 @@
+import {
+  FETCH_LAB_CONCEPTS_SUCCESS,
+  FETCH_LAB_CONCEPTS_FAILURE,
+} from '../actionTypes';
+import axiosInstance from '../../config';
+import loading from '../loading';
+
+export const fetchLabConceptsSuccess = response => ({
+  type: FETCH_LAB_CONCEPTS_SUCCESS,
+  payload: response,
+});
+
+export const fetchLabConceptsFailure = error => ({
+  type: FETCH_LAB_CONCEPTS_FAILURE,
+  payload: error,
+});
+
+export const fetchLabConcepts = conceptUUID => (dispatch) => {
+  dispatch(loading('FETCH_LAB_CONCEPTS', true));
+  return axiosInstance.get(`/concept/${conceptUUID}`)
+    .then((response) => {
+      if (response.status !== 200) {
+        throw Error(response.statusText);
+      }
+      dispatch(loading('FETCH_LAB_CONCEPTS', false));
+      return response;
+    })
+    .then(response => dispatch(fetchLabConceptsSuccess(response)))
+    .catch((error) => {
```

Comment:
why are you stacking another `.then` when you can do what you are doing inside it in the first `.then` block

Reply:
Good catch

Reply:
Yes, you are right.
I would ensure to update this block of code.